internal/wasm/store_test.go

```diff
@@ -182,6 +182,38 @@ func TestStore_CloseModule(t *testing.T) {
 	}
 }
 
+func TestStore_CloseStore(t *testing.T) {
```

Comment:
maybe make this two tests.. ex `t.Run("nothing closed", func...` and `t.Run("partially closed", func...` where the latter closed the module importing but not the imported. this would be the case if someone forgot to close wasi